{
    "id": "bundle--ad1ee982-dfcc-4d03-adf9-970c7609ddd3",
    "objects": [
        {
            "created": "2021-03-13T20:10:04.60445Z",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "description": "The get_image_dimensions function in the image-handling functionality in Django before 1.3.2 and 1.4.x before 1.4.1 uses a constant chunk size in all attempts to determine dimensions, which allows remote attackers to cause a denial of service (process or thread consumption) via a large TIFF image.",
            "external_references": [
                {
                    "external_id": "CVE-2012-3444",
                    "source_name": "cve"
                }
            ],
            "id": "vulnerability--53dc1e0b-d5d9-4d69-8342-1b6ee807c534",
            "modified": "2021-11-22T14:54:52.900149Z",
            "name": "CVE-2012-3444",
            "spec_version": "2.1",
            "type": "vulnerability"
        }
    ],
    "type": "bundle"
}